script/dev-deploy

```diff
@@ -367,11 +369,8 @@ class ZookeeperService(Service):
             if is_container_running(host, container_name, container_bin):
                 logwarn(f"Service {self.KEYS[0]} already running.")
             else:
-                mount_zk_conf = ""
-                if zk_remote_conf is not None:
-                    mount_zk_conf = f"-v {zk_remote_conf}:/conf/zoo.cfg"
                 enable_metrics_provider = (
-                    '-e ZOO_CFG_EXTRA="metricsProvider.className=org.apache.zookeeper.metrics.prometheus.PrometheusMetricsProvider"'
+                    '-e ZOO_CFG_EXTRA="metricsProvider.className=org.apache.zookeeper.metrics.prometheus.PrometheusMetricsProvider $ZOO_CFG_EXTRA"'
```

Comment:
I don't think it's necessary to add `$ZOO_CFG_EXTRA` here because
  1. we always use the default configuration file now and do not support users to upload their configuration files
  2. we don't support adding arbitrary configuration parameters because we are hard-coding the commands here
 
So, the `ZOO_CFG_EXTRA` parameter is deterministic and will never change.

Reply:
fixed. thx